automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/builddeploy/BuildDeploySpec.scala

```diff
@@ -38,6 +39,23 @@ class BuildDeploySpec extends FreeSpec with WebBrowserSpec with Matchers with La
 
     }
 
+    // this test ensures the build outputs a git commit hash value. As a side effect, this test also
+    // logs the value it finds, so we have a record - albeit buried - of the exact codebase on which
+    // the test ran.
+    "should populate a commithash value in HTML source" in {
+      withWebDriver { implicit driver =>
+        driver.get(s"${FireCloudConfig.FireCloud.baseUrl}index.html")
+        val commithash = driver
+          .findElement(By.xpath("//meta[@name='application-name']"))
+          .getAttribute("data-commithash")
+
+        logger.info(s"unit test commithash: $commithash")
+
+        commithash shouldNot be(empty)
```

Comment:
any need for checking `commithash` is formatted correctly and use `eventurally` if it's page rendering time-sensitive?

Reply:
page rendering is not time-sensitive; this is populated during the build; by the time it's served to this test, it's a static file. I'm also intentionally not checking format (beyond non-empty) to allow for future changes in format.